src/containers/PrimeStatusBanner/index.tsx

```diff
@@ -137,11 +142,18 @@ export const PrimeStatusBannerUi: React.FC<PrimeStatusBannerUiProps> = ({
   return (
     <Card
       className={cn(
-        'flex flex-col content-center md:flex-row md:justify-between',
+        'relative flex flex-col content-center md:flex-row md:justify-between',
         isUserXvsStakeHighEnoughForPrime && 'items-start md:items-center',
+        shouldShowPrimeTokensLeftIndicator && 'mt-[12.5px]',
```

Comment:
That's a very small thing: given you've applied a negative top margin of 3rem to the div wrapping `PrimeTokensLeft`, you can apply a top margin of 3rem here as well:
```suggestion
        shouldShowPrimeTokensLeftIndicator && 'mt-3',
```